README.md

```diff
@@ -56,3 +56,9 @@ function App() {
 
 export default App;
 ```
+
+To use Answers React Components stylesheet, import /bundle.css as follows:
```

Comment:
I would say something about how this is an alternative to using tailwind directly